sdks/go/pkg/beam/core/runtime/exec/plan.go

```diff
@@ -131,6 +141,44 @@ func (p *Plan) Execute(ctx context.Context, id string, manager DataContext) erro
 	return nil
 }
 
+// Runs any callbacks registered by the bundleFinalizer. Should be run on bundle finalization.
```

Comment:
Nit: Godoc comments should start with the name of the function. (See https://go.dev/blog/godoc)
```suggestion
// Finalize runs any callbacks registered by the bundleFinalizer. Should be run on bundle finalization.
```